BMS_ChargeRateCheck.c

```diff
@@ -0,0 +1,43 @@
+#include "BMS_BatteryCheck.h"
+
+BatteryData_t ChargeRate_Specification = {
+  0.0,            
+  0.8, 
+  0.04,
+  0.76,
+  "ChargeRate" 
+  };
+  
+/**
+ ***************************************************************************************************
+ * Function Name: IschargeRateStable
+ * 
+ * Function Description: Checks the ChargeRate is within the range of permissible working limit.
+ *
+ * \param  float chargeRate
+ *         
+ * \return  Integer
+ * \retval  BATTERY_HEALTH_OK or BATTERY_HEALTH_NOT_OK indicating Charge rate status.
+ ***************************************************************************************************
+ */
+ 
+int IschargeRateStable(float chargeRate)

```

Comment:
This function is very similar to the functions in the other two files. If this function accepts a `BatteryData_t`, we can make one function instead of three. Try it out.